pages/deposit-dates.css

```diff
@@ -1,46 +1,47 @@
-/* TODO: Custom media from layout.css doesn't work */
-@custom-media --screen-until-desktop (max-width: 727px);
-
-.table thead > :first-child {
-  border-top-width: 2px;
+%item {
+  flex: 3 3 90%;
+  margin: 0 var(--component-spacer-x, 1rem) var(--component-spacer-y, 1.5rem);
```

Comment:
Does this variable even exist? Can you please always explicitly add TODO comment when something should be created.